blog/2023/12-10-android-tv-16/index.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Android TV version 0.16
+authors:
+  - nielsvanvelzen
+slug: androidtv-v0.16.0
+tags: [release, android-tv]
+---
+
+It has been a busy year with a lot of work on the Android TV app. After an extended beta period today is finally the day to share this work
+to the public. Read along to learn everything about improved music playback, the new screensaver and more!
+
+\- Niels
```

Comment:
I wouldn't bother with the sign here since your name is at the top implying you wrote it all, but I won't object ;-)

Reply:
You started it in the [very first blog post](https://jellyfin.org/posts/new-website) when we launched the new website! I've done this in most of my recent blog posts now so I'll keep doing it :p

Reply:
Ditto